src/osd/OSD.cc

```diff
@@ -10297,6 +10312,16 @@ bool OSD::unsupported_objstore_for_qos()
                    store->get_type()) != unsupported_objstores.end();
 }
 
+std::string_view OSD::osd_scheduler_type()
```

Comment:
Since we're accessing this without a lock, it needs to be const.  It follows that get_scheduler_type must be as well.

Reply:
Done.